packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -627,11 +629,6 @@ Information about project "Runner":
     expect(XcodeProjectInfo.expectedBuildConfigurationFor(BuildInfo.release, 'Runner'), 'Release');
   });
 
-  testWithoutContext('expected scheme for flavored build is the title-cased flavor', () {
```

Comment:
What happened to this test?

Reply:
this test was removed because the helper function it tests (expectedSchemeFor) was removed.

Reply:
I more meant I don't see an equivalent test for your `schemeFor` that is checking it's using title case.